### PR TITLE
added category pages

### PR DESCRIPTION
**Pull request by b0llu**
image
image
image

### CODE REVIEW

File: landing.css [hunk 1]
```diff
@@ -28,70 +28,51 @@ a {
...
-    margin-right: 2rem;
-    padding: 5px 2rem;
-}
-
-.quiz-item .card-title {
-    font-size: 1.5rem;
-    font-weight: 500;
-    /* margin-top: auto; */
+    margin-top: 2rem;
+    background-color: var(--for-quiz);
```
Comment (Anurag021) [inline landing.css:52]: just a suggestion on this variable name, it would make more sense if it would be something like --quiz-color, rest looks good.

**Review by Anurag021 — APPROVED** [review]
Code looks good, just one suggestion above and rest all good to go.

File: landing.css (hunk 1 above)
Comment (b0llu) [reply landing.css:52]: thanks for pointing it out i will think for a new name

File: category-page/category-topic-one.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <link rel="stylesheet" href="../quiz-css/grid-layout.css" />
+    <title>StormIZ</title>
+  </head>
+  <body>
+    <div class="quiz-container">
+      <nav>
+        <a href="">
+          <i class="theme fab fa-twitter"></i>
+        </a>
+        <a href="">
```
Comment (Developers-codz) [inline category-page/category-topic-one.html]: add links

File: category-page/category-topic-two.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        <i id="toggle-theme" class="theme fas fa-moon"></i>
+      </nav>
+      <section>
+        <h2 class="subtitle">Movie Quiz</h2>
+        <div class="quiz-item">
+                <div class="card-container product-card card-shadow">
+                    <div class="quiz-img">
+                        <img class="card-img"
+                            src="https://www.iwmbuzz.com/wp-content/uploads/2020/05/10-secret-facts-of-avengers-endgame-you-should-know-920x518.jpg"
+                            alt="..." />
```
Comment (Developers-codz) [inline category-page/category-topic-two.html]: provide meaningful value to alt

**Review by Developers-codz — APPROVED** [review]
Everything else looks good to me

File: category-page/category-topic-one.html (hunk 2 above)
Comment (b0llu) [reply category-page/category-topic-one.html]: thanks for pointing it out

File: category-page/category-topic-two.html (hunk 3 above)
Comment (b0llu) [reply category-page/category-topic-two.html]: thanks for pointing it out

**Pull request merged**